controllers/article_id.js

```diff
@@ -0,0 +1,18 @@
+const { fetchArticleById } = require("../models/article_id");
+
+exports.getArticleById = (req, res, next) => {
+  const articleId = req.params.article_id;
+  if (isNaN(articleId)) {
+    return res.status(400).json({ message: "Invalid article ID" });
+  }
```

Comment:
I like what you've done here with you use of `isNan` to check if the ID is invalid, I would think however that it might show off your SQL skills a little more if you allow the invalid ID to go into the query - as you're using `$1` for it SQL will throw it back as an error for you. You can then take that SQL error and handle it in your `app.js` file inside your error handler

Reply:
SQL errors all have a `code` property unique to them, that is something you can check in your error handler